gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -450,6 +450,8 @@ def update_policy_target_added(self, context, policy_target):
                 network_function_id = network_function_map.network_function_id
                 self.nfp_notifier.policy_target_added_notification(
                     context.plugin_context, network_function_id, policy_target)
+                self._wait_for_network_function_operation_completion(
```

Comment:
In general in SO and DO, is the state transition not done based on current state?